cmd/podman/varlink/io.projectatomic.podman.varlink

```diff
@@ -53,32 +53,72 @@ type ImageSearch (
     star_count: int
 )
 
+# ListContainer is the returned struct for an individual container
+type ListContainerData (
+    id: string,
+    image: string,
+    imageid: string,
+    command: []string,
+    createdat: string,
+    runningfor: string,
+    status: string,
+    ports: [][string]string,
```

Comment:
Can we either do arrays of structs, or somehow expose the keys we expect to be in every map? So people know how to parse the ports and mounts?

Reply:
i took a todo from our irc argument about this.